### PR TITLE
Seed refactor add techhire location model

### PR DESCRIPTION
**Pull request by Jwan622**
-seed file changed
-added TechhireLocation model
-added video model tests
-moved geocoding to the TechhireLocation Model from the company model.

### CODE REVIEW

File: app/assets/stylesheets/home/index.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 path {
   stroke: grey;
   stroke-width: 1.00px;
-  fill: #f8f4e8;
+  fill: #a88888;
```
Comment (cluhring) [inline app/assets/stylesheets/home/index.css:4]: make it gray?

File: app/models/company.rb [hunk 2]
```diff
@@ -14,17 +12,5 @@ class Company < ActiveRecord::Base
   scope :confirmed, -> { where status: "confirmed"}
   scope :dead, -> { where status: "dead"}
 
-  geocoded_by :full_address
```
Comment (cluhring) [inline app/models/company.rb:17]: thanks for taking this out Jwan.  no more broken tests!  fingers crossed.

File: app/models/techhire_location.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+class TechhireLocation < ActiveRecord::Base
```
Comment (cluhring) [inline app/models/techhire_location.rb:1]: nice!

File: db/seeds.rb [hunk 4]
```diff
@@ -1,244 +1,125 @@
...
 pdf.save
 
-# Cities
-Company.create(
-  name: "company",
-  organization: "org",
-  title: "title",
-  state: "Kentucky",
+# techhire_locations
+TechhireLocation.create(
```
Comment (cluhring) [inline db/seeds.rb:7]: Cool - looking forward to seeing your new and improved map!

File: spec/factories/techhire_locations.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
```
Comment (cluhring) [inline spec/factories/techhire_locations.rb:1]: do we want this?

File: spec/models/techhire_location_spec.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe TechhireLocation, type: :model do
```
Comment (cluhring) [inline spec/models/techhire_location_spec.rb:3]: test to check on geocoder?

File: spec/models/video_spec.rb [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+  it "should be invalid with a blank url" do
+    video = build(:video, url: "")
+    video.save
+
+    expect(video.errors.full_messages.to_sentence).to eql("Error Please enter a valid url.")
+    expect(video).to be_invalid
+  end
+
+  it "should ve valid with a lot of 1,2,3 or 4" do
```
Comment (laurawhalin) [inline spec/models/video_spec.rb:18]: small typo

File: spec/models/video_spec.rb [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+    video1 = build(:video, slot:1)
+    video2 = build(:video, slot:2)
+    video3 = build(:video, slot:3)
+    video4 = build(:video, slot:4)
```
Comment (laurawhalin) [inline spec/models/video_spec.rb:22]: Maybe change one of these so it doesn't have a slot, just to show that it is still a valid video?

File: app/assets/stylesheets/home/index.css (hunk 1 above)
Comment (Jwan622) [inline app/assets/stylesheets/home/index.css:4]: I was trying to fix the map color.

File: app/models/company.rb (hunk 2 above)
Comment (Jwan622) [inline app/models/company.rb:17]: company is refactored now.

File: spec/factories/techhire_locations.rb (hunk 5 above)
Comment (Jwan622) [inline spec/factories/techhire_locations.rb:1]: yes. will use this in tests later.

File: spec/models/techhire_location_spec.rb (hunk 6 above)
Comment (Jwan622) [inline spec/models/techhire_location_spec.rb:3]: sigh.

File: spec/models/video_spec.rb (hunk 7 above)
Comment (Jwan622) [inline spec/models/video_spec.rb:18]: sigh.

File: app/models/video.rb [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+  def verify_youtube_video
+    yt_regex = /www\.youtube\.com\/watch\?v=([a-zA-Z0-9_-]*)/
+    
+    if self.url.blank?
+      errors[:error] << "Please enter a valid url."
+    elsif !self.url.match(yt_regex)
+      errors[:error] << "Please check your YouTube format."
+    end
+  end
+end
```
Comment (scottalexandra) [inline app/models/video.rb:27]: Great! Glad we have a model now!

**Comment by scottalexandra** [comment]
All Tests Passing!

**Pull request merged**